product_docs/docs/pem/10/debugger.mdx

```diff
@@ -1,136 +1,129 @@
 ---
-title: "PL Debugger"
-navTitle: "Using PL Debugger"
+title: "PL debugger"
+navTitle: "Using the PL debugger"
 ---
 
 ![Debugger page - Parameters tab](images/debug_main.png)
 
-The debugger may be used to debug PL/pgSQL functions in PostgreSQL, as
-well as EDB-SPL functions, stored procedures and packages in Advanced
-Server. The Debugger is available as an extension for your PostgreSQL
-installation, and is distributed as part of Advanced Server. You must
-have superuser privileges to use the debugger.
+You can use the debugger to debug PL/pgSQL functions in PostgreSQL,
+EDB-SPL functions, stored procedures, and packages in EDB Postgres Advanced
+Server. The debugger is available as an extension for your PostgreSQL
+installation and is distributed as part of EDB Postgres Advanced Server. You need
+superuser privileges to use the debugger.
 
-Before using the debugger, you must modify the `postgresql.conf` file,
-adding the server-side debugger components to the the value of the
+Before using the debugger, modify the `postgresql.conf` file,
+adding the server-side debugger components to the value of the
 `shared_preload_libraries` parameter:
 
+```
 shared_preload_libraries =\'\$libdir/`other_libraries`/plugin_debugger\'
+```
 
 After modifying the `shared_preload_libraries` parameter, restart the
 server to apply the changes.
 
-The debugger may be used for either in-context debugging or direct
+You can use the debugger for in-context debugging or direct
 debugging of a target function or procedure. When you use the debugger
 for in-context debugging, you set a breakpoint at the first line of a
-program; when a session invokes the target, control is transferred to
+program. When a session invokes the target, control is transferred to
 the debugger. When using direct debugging, the debugger prompts you for
-any parameters required by the target, and then allows you to step
+any parameters required by the target and then allows you to step
 through the code.
 
-## In-context Debugging
+## In-context debugging
 
 To set a breakpoint at the first line of a program, right-click the name
-of the object you would like to debug, and select `Set breakpoint` from
-the `Debugging` sub-menu. The debugger window will open, waiting for
-another session to invoke the program.
+of the object you want to debug, and select **Debugging > Set breakpoint**. 
+The debugger window opens and waits for another session to invoke the program.
```

Comment:
While I left screen captures that seemed to be imparting some information, I removed some that were just showing the interface. Consider whether the remaining ones are adding value.